tests/ui_tools/test_messages.py

```diff
@@ -1968,6 +1968,7 @@ def test_footlinks_limit(self, maximum_footlinks, expected_instance):
     def test_mouse_event_left_click(
         self, mocker, msg_box, key, widget_size, compose_box_is_open
     ):
+        expected_keypress = primary_key_for_command("ACTIVATE_BUTTON")
```

Comment:
Needed to edit this test as mouse_event always calls the `primary_key_for_command("ACTIVATE_BUTTON")` and does not use the passed key.